backend/src/org/commcare/suite/model/Profile.java

```diff
@@ -84,6 +88,21 @@ public Profile(int version, String authRef) {
     public int getID() {
         return recordId;
     }
+    
+	/*
+	 * Return the uniqueId assigned to this app from HQ
+	 */
+	public String getUniqueId() {
+		return this.uniqueId;
+	}
+	
+	/*
+	 * Return whether this Profile was generated from an old version of profile.ccpr
+	 * (i.e. does not have a uniqueId or displayName)
+	 */
+	public boolean fromOld() {
```

Comment:
Good that this is commented, but it's pretty vague. How about returning !fromOld and calling it multipleAppsEnabled?